scripts/openzfs-tracking.sh

```diff
@@ -71,12 +72,10 @@ DESCRIPTION:
 OPTIONS:
 	-h		Show this message
 	-d directory	Git repo with openzfs and zfsonlinux remotes
-	-e exceptions	Exception file
 
 EXAMPLE:
 
 $0 -d ~/openzfs-tracking/zfs \\
-    -e ~/zfs-buildbot/scripts/openzfs-exceptions.txt \\
```

Comment:
It would be nice to preserve the ability to supply a local markdown file.  It's fine if it defaults to pulling from the wiki when the `-e` option isn't supplied.